libsodium-sys/libsodium

```diff
@@ -0,0 +1 @@
+Subproject commit f01d7c128214e8f976889c8093a42d9cae1abe4b
```

Comment:
I'll just hold off on the merge until 1.0.11 is released. Thanks for all
your work.
On Thu, 28 Jul 2016 at 11:48, Jake McGinty <notifications@github.com> wrote:

> In libsodium-sys/libsodium
> <https://github.com/dnaq/sodiumoxide/pull/135#discussion_r72593772>:
>
> > @@ -0,0 +1 @@
> > +Subproject commit f01d7c128214e8f976889c8093a42d9cae1abe4b
>
> Ahh, I didn't know about what was coming in 1.0.11. Nifty!
>
> I'd say from the conservative perspective, using specific stable releases
> whenever possible in cryptography libraries makes me do less of a
> double-take though (and it looks like the author signs releases, but not
> individual commits). I suppose this will be a non-issue once 1.0.11 is
> released officially.
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/dnaq/sodiumoxide/pull/135/files/cf42804a26463f8adb595529a33d1491c77e3a50#r72593772>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABPVBT-f-sBB-xtaTnuG8G30HSl4mQruks5qaHrSgaJpZM4JJZI9>
> .
>